src/store/socket.js

```diff
@@ -788,6 +810,9 @@ export default store => {
       case "players/update":
         session.sendPlayer(payload);
         break;
+      case "players/setPronoun":
```

Comment:
If you use `players/update` for changing the pronouns, you need to adjust the switch case above accordingly and check for the property that gets updated.